### PR TITLE
Prod iam credentials [IAM-185]

### PR DESCRIPTION
**Pull request by Nesze**

### CODE REVIEW

File: dev-aws/kafka-shared/iam_credentials.tf [hunk 1]
```diff
@@ -5,8 +5,8 @@ resource "kafka_topic" "iam_credentials_v1" {
   config = {
     # retain 100MB on each partition
     "retention.bytes" = "104857600"
-    # keep data for 7 days
-    "retention.ms" = "604800000"
+    # keep data for 60 days
+    "retention.ms" = "5184000000"
```
Comment (Nesze) [inline dev-aws/kafka-shared/iam_credentials.tf:9]: Retaining events for two months, but they are quite low-volume.

Comment (sbuliarca) [reply dev-aws/kafka-shared/iam_credentials.tf:9]: Do we need the same in prod~?

Comment (Nesze) [reply dev-aws/kafka-shared/iam_credentials.tf:9]: Oh boy, we only need it in prod 😅

**Comment by rcrowe** [comment]
@sbuliarca what do you want to do with team/file naming?
This PR introduces a new file, `iam.tf` already exists.

**Comment by sbuliarca** [comment]
> @sbuliarca what do you want to do with team/file naming? This PR introduces a new file, `iam.tf` already exists.

Indeed, I didn't pay attention to that. We can keep them in one file or split them. As long as it has the team prefix, it should be ok, but we need to keep them in one folder

**Review by sbuliarca — APPROVED** [review]

**Pull request merged**